app/main.py

```diff
@@ -1,8 +1,40 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def set_wife(self, wife: str) -> None:
+        self.wife = wife
+
+    def set_husband(self, husband: str) -> None:
+        self.husband = husband
+
+
+def create_person_list(people_list: list) -> list:
+    result = []
+
+    for person_dict in people_list:
+        name = person_dict["name"]
+        age = person_dict["age"]
+        person_object = Person(name, age)
+        result.append(person_object)
```

Comment:
```suggestion
        result.append(Person(name, age))
```
avoid creating extra variables